CONTRIBUTING.md

```diff
@@ -19,3 +19,19 @@ Allowed types:
 - **test**: Adding missing tests or correcting existing tests
 
 _(based on [Angular contribution guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type))_
+
+## Release a new version
```

Comment:
This workflow looks far from automated.
Despite not having a monorepo, checkout the [lerna publish workflow](https://github.com/lerna/lerna/tree/master/commands/publish#readme) to get an idea of a automated release

Reply:
I agree that this is not 100% automated, but IMO, we should not introduce a 100% automation for this at the moment, without testing it with this approach for a while. Specifically an automated git push to remote can really complicate things, when stuff has to be reverted. I would always check the automated changelog file before pushing, just to see if the formatting is correct. Otherwise this could lead to a mis-formatted markdown file, which is not ideal for the users.

Please bear in mind, that we don’t add the changelog entries manually alongside each pull request, where the formatting would be under control of the review process. When the creation of the changelog itself is done **during** the release process, we have no chance of controlling the formatting of that and there is a chance that we need to redo the changes, reverting thins in git and do the whole thing manually anyway.

If you look to the list of tasks to do for a new release, steps 1–3 and 8–10 have to be done manually anyway.